pathod/protocols/websockets.py

```diff
@@ -53,4 +53,4 @@ def handle_websocket(self, logger):
                         )
                         lg("crafting websocket spec: %s" % frame_log["spec"])
                         self.pathod_handler.addlog(frame_log)
-        return self.handle_websocket, None
+        return self.handle_websocket, None  # pragma: no cover
```

Comment:
The `while True` never returns, so this return statement never triggers. I guess it can be removed completely, but it might be required for type deduction.